config/routes.rb

```diff
@@ -183,7 +183,10 @@
 
     resources :ownership_calls, only: :index
     resources :webauthn_credentials, only: :destroy
-    get 'webauthn_verification/:webauthn_token', to: 'webauthn_verifications#prompt', as: 'webauthn_verification'
+    resource :webauthn_verification, only: [] do
+      get ':webauthn_token', to: 'webauthn_verifications#prompt', as: ''
+      post ':webauthn_token', to: 'webauthn_verifications#authenticate', as: :authenticate, constraints: { format: /json/ }
```

Comment:
Why a regex for `json`?

Reply:
I did some copy and pasta with the regex from another route. I'm intending to keep it as is since we're planning to support html as a format too -> `format: /html|json/`.